ci-operator/config/stolostron/search-e2e-test/stolostron-search-e2e-test-main.yaml

```diff
@@ -72,6 +72,16 @@ tests:
       IMAGE_REPO: search-e2e-test
       REGISTRY_ORG: stolostron
     workflow: ocm-ci-image-mirror
+- as: latest-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-e2e-test
+    env:
+      IMAGE_REPO: search-e2e-test
+      IMAGE_TAG: latest
```

Comment:
Do we use the latest tag for testing?

Reply:
The latest tag is being used for the image that is published to quay.